src/cycles/add-cycles-back.ts

```diff
@@ -0,0 +1,79 @@
+import * as _ from 'lodash';
```

Comment:
NOOOOOOO!!! 😢 

Please don't use `lodash`.. at all if you can help it, otherwise please import the exact lodash module you need to use.

You should notice that `lodash` is not a dependency fo this package - that is intentional.